inst/tinytest/test_url_parsing.R

```diff
@@ -23,7 +25,24 @@ for(i in which(grepl("embl", names(paths))) ) {
   expect_equal(parsed$region,   "auto")
 }
 
+for(i in which(grepl("eopf_path_1", names(paths))) ) {
+  expect_silent(parsed <- .url_parse_other(paths[[ i ]]))
+  expect_equal(parsed$bucket,   "e05ab01a9d56408d82ac32d69a5aae2a:202505-s02msil2a")
+  expect_equal(parsed$hostname, "https://objectstore.eodc.eu:2222")
+  expect_equal(parsed$object,   "17/products/cpm_v256/S2A_MSIL2A_20250517T085541_N0511_R064_T35QKA_20250517T112203.zarr/")
+  expect_equal(parsed$region,   "auto")
+}
+
+for(i in which(grepl("eopf_path_2", names(paths))) ) {
+  expect_silent(parsed <- .url_parse_other(paths[[ i ]]))
```

Comment:
tinytest, as opposed to testthat, doesn't rely on `pkg::load_all()` so internal functions have to be prefixed by `:::`:

```suggestion
  expect_silent(parsed <- Rarr:::.url_parse_other(paths[[ i ]]))
```